src/less/_home.less

```diff
@@ -0,0 +1,23 @@
+.order-view {
+  display: flex;
+  flex-flow: column;
+  /* justify-content: space-between; */
```

Comment:
remove/delete if not needed

Reply:
Thank you for your review all changes have been made.